src/mod_vcard.erl

```diff
@@ -414,7 +415,13 @@ remove_user(Acc, User, Server) ->
 remove_user(User, Server) ->
     LUser = jid:nodeprep(User),
     LServer = jid:nodeprep(Server),
-    mod_vcard_backend:remove_user(LUser, LServer).
+    lists:foreach(fun(B) ->
+        try
+            B:remove_user(LUser, LServer)
+        catch
+            _:_ ->
+                ok
```

Comment:
We need to log warning here, even if it's false positive. Just in case.

Reply:
ok